README.md

```diff
@@ -9,7 +9,17 @@ This library requires Spark 1.3+, Scala 2.10+, RabbitMQ 3.5+
 
 ## Using the library
 
-For using RabbitMQ-Receiver library you have to clone the repo an install the jar in your maven local repo by doing:
+There are two ways of using RabbitMQ-Receiver library:
+
+You can link against this library by putting the following lines in your program:
```

Comment:
I suggest:

"*To add the next dependency inside of your pom.xml" 
<dependency>
blahblahblah
</dependency>

instead of  "You can ..."